hw_oop/ex1.py

```diff
@@ -0,0 +1,31 @@
+class Shape:
+
+    def __init__(self, w, h):
```

Comment:
Here you have space before init, in other classes don't. Leave only one version, for codestyle to be solid.